dashboard/config.py

```diff
@@ -2,11 +2,10 @@
 from pprint import pprint
 import textwrap
 from typing import List
-import pathlib
 
 from dotenv import dotenv_values
 
-from .config_file import ConfigFile
+from dashboard.config_file import ConfigFile
```

Comment:
nit: Should this be CAPS_LOCK instead of PascalCase?

Reply:
Why would this be CAPS_LOCK? It's a class